### ISSUE TITLE
Locize-cli timeouts on sync

### ISSUE THREAD

**Issue by vcarnogu** [locize/locize-cli#86]
## 🐛 Bug Report

Timeout on `locize sync` command.

## To Reproduce

Wa have 15 languages, 85.000 words total.
`locize sync --path ./public/locales --project-id 298secret338 --api-key a98secret3 --reference-language-only false`

I was able to synchronize it twice but most of this time I get an error like this:

```
FetchError: request to https://api.locize.app/2fc23692-22e1-4cf3-8425-60354bc718e8/latest/ru/generalDialog?ts=1690548066168 failed, reason: connect ETIMEDOUT 13.32.110.68:443
    at ClientRequest.<anonymous> (/Users/usrname/Documents/project/project-web/node_modules/locize-cli/node_modules/node-fetch/lib/index.js:1505:11)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:490:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) 
```

If will fails on various languages and namespaces (see URL above)

## Expected behavior

Synchronisation should pass.

## Your Environment

- *runtime version*: Node 19.2.0
- *Locize CLI*: >=locize-cli: ^7.14.6
- *os*: Mac

**Comment by adrai** [comment]
Hi Vaclav,

We just tried to reproduce it several times in different networks with your translations, but we were not able to trigger a timeout.
Depending on your local network setup, the problem may be in proxy or firewall restrictions (by company).

Nevertheless, we've just released a new CLI version v7.14.7 that will retry the request in case of a ETIMEDOUT error.

I hope this helps.

**Comment by vcarnogu** [comment]
Thanks @adrai. I will give it a try. In the meantime a figured out that when I limit languages with `--language [lc]` it doesn't time out.
